internal/rego/rego.go

```diff
@@ -124,9 +129,13 @@ func loadRegoFiles(files []string) ([]File, error) {
 	return regoFiles, nil
 }
 
-func getModuleRulesActions(module *ast.Module) []string {
+func getModuleRulesActions(module *ast.Module) ([]string, error) {
+	re, err := regexp.Compile(`^\s*([a-z]+)\s*\[\s*msg`)
+	if err != nil {
+		return nil, fmt.Errorf("parse rule: %w", err)
```

Comment:
Error message isn't quite right. It's about compiling a regex, not parsing the rego rules

Reply:
Yeah, that's fair. I'll update the message.

Reply:
All set!